packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,403 @@
+            	"include": [],
+            	"exclude": []
+        	},
+        	"groups": {
+            	"include": [
+                	"EVERYONE"
+            	],
+            	"exclude": []
+        	}
+    	},
+    	"grantTypes": {
+        	"include": [
+            	"implicit",
+            	"password",
+				"client_credentials",
+            	"authorization_code",
+            	"urn:ietf:params:oauth:grant-type:token-exchange"
+        	]
+    	},
+    	"scopes": {
+        	"include": [
+            	"*"
+        	]
+    	}
+	},
+	"actions": {
+    	"token": {
+        	"accessTokenLifetimeMinutes": 60,
+        	"refreshTokenLifetimeMinutes": 0,
+        	"refreshTokenWindowMinutes": 10080
+    	}
+	},
+	"type": "RESOURCE_ACCESS"
+}'
+```
+
+## Configure the token exchange grant type for client 2
+
+In this request, use the client ID for client 2 that you created in the "Set up the application" section. Copy the response that is returned from this request for use in the next step.
+
+**Request example**
+
+```bash
+  curl --request GET \
+  --url https://${yourOktaDomain}/oauth2/v1/clients/{clientId}' \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: SSWS <apiKey>'
+```
+
+### Update client 2 with the token exchange grant
+
+In this request, update client 2 with the token exchange grant. Use the response from the last step to create your UPDATE request. You need to update the `grantTypes` parameter by adding the value `urn:ietf:params:oauth:grant-type:token-exchange` so that the token exchange is an allowed grant type for the client.
+
+> **Note:** All clients that want to leverage Native SSO and SLO must be configured with this grant type.
+
+```json
+  curl --location --request PUT \
+  --url https://${yourOktaDomain}/oauth2/v1/clients/{clientId} \
+  --header 'Content-Type: application/json' \
+  --header 'Accept: application/json' \
+  --header 'Authorization: SSWS <apiKey>' \
+  -d '{
+      "client_id": "{clientId}",
+      "client_id_issued_at": 1618351934,
+      "client_name": "Target Client",
+      "client_uri": null,
+      "logo_uri": null,
+      "redirect_uris": [
+        "https://httpbin.org/get"
+    ],
+    "response_types": [
+       "code"
+    ],
+    "grant_types": [
+        "authorization_code",
+        "refresh_token",
+        "urn:ietf:params:oauth:grant-type:token-exchange"
+    ],
+    "token_endpoint_auth_method": "none",
+    "application_type": "native"
+}'
+```
+
+## Create a Native desktop session lifetime
+
+With Native SSO, you can create a separate forked session and associate it with the device secret. The device secret assumes the lifetime of the first refresh token that it was minted with. The device secret has the same idle time and maximum time according to the authorization server policy through which it was minted. From there, the device secret and refresh token idle lifetimes are independent of each other.
```

Comment:
With Native SSO, you can create a separate forked session and associate it with the device secret. --This is an implementation detail.

Just start from.....The device secret assumes the lifetime...

Reply:
Fixed.